rio_tiler/reader.py

```diff
@@ -531,6 +535,11 @@ def point(
     """
     indexes = cast_to_sequence(indexes)
 
+    if buffer and buffer % 0.5:
+        raise InvalidBufferSize(
+            "`buffer` must be a multiple of `0.5` (e.g: 0.5, 1, 1.5, ...)."
+        )
```

Comment:
we use the same definition for buffer (floating point, multiple of 0.5) from the other methods